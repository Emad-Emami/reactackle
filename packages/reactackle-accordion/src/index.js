import Accordion from './Accordion';
import AccordionItem from './AccordionItem/AccordionItem';

export { Accordion, AccordionItem };
