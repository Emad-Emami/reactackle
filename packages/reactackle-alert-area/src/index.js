import AlertArea from './AlertArea';

export { AlertArea };
