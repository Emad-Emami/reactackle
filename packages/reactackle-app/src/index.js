import App from './App';

export { App };
export * from './BottomRegion';
export * from './TopRegion';
export * from './MainRegion';
