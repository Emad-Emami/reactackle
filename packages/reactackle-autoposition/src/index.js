import AutoPosition, { recalculateAllAutoPosition } from './AutoPosition';

export { AutoPosition, recalculateAllAutoPosition };
