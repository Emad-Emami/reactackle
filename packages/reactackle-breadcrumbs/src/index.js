import Breadcrumbs from './Breadcrumbs';

export { Breadcrumbs };
export * from './BreadcrumbsItem';
