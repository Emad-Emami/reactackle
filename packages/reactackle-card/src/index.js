import Card from './Card';

export { Card };
export * from './CardActions';
export * from './CardAreaMain';
export * from './CardAreaSide';
export * from './CardContent';
export * from './CardHeaderPrimary';
export * from './CardHeaderSecondary';
export * from './CardMedia';
