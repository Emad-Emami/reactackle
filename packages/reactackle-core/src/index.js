// theme
// defaultTheme
// composeReactackleTheme
// defaultThemeBlueprintBroadcast
// registerDefaultComponentTheme ++
// extractThemeOrDefault ++

// styleHelpers - for combining local theme
// export everything

// styleFunctions
// export everything

// - resetStyles
// - globalStyles
// - heladerClasses

// base
// export nothing

// constants
// export everything

// mixins
// export everything

// import {
//   registerDefaultComponentTheme,
//   extractThemeOrDefault,
// } from './theme';

// export {
//   registerDefaultComponentTheme,
//   extractThemeOrDefault,
// };

export * from './theme';
export * from './mixins';
export * from './styleFunctions';
export * from './styleHelpers';
export * from './utils';
export * from './constants/constants';
export * from './hoc/withExternalProps';
export * from './globalStyles';
export * from './resetStyles';
export * from './Theme/Theme';
export * from './Theme/withTheme';
