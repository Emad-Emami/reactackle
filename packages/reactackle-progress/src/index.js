import { registerDefaultComponentTheme } from 'reactackle-core';
import componentTheme from './styles/theme';

registerDefaultComponentTheme('progress', componentTheme);

export * from './CircleProgress/CircleProgress';
export * from './LinearProgress/LinearProgress';
export * from './Preloader/Preloader';
