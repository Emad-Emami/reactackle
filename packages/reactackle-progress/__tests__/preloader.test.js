import React from 'react';
import renderer from 'react-test-renderer';

import { Preloader } from '../src';

jest.mock('react-dom');

describe('<Preloader />', () => {
  it('renders correctly with alignX prop set to one of enum values', () => {
    const tree = renderer.create(
      <Preloader kind='linear' alignX='left' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with alignY prop set to one of enum values', () => {
    const tree = renderer.create(
      <Preloader kind='linear' alignY='top' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with height prop set to String', () => {
    const tree = renderer.create(
      <Preloader kind='linear' height='Some string' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with kind prop set to one of enum values', () => {
    const tree = renderer.create(
      <Preloader kind='linear' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with width prop set to Sring', () => {
    const tree = renderer.create(
      <Preloader kind='linear' width='Some string' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });
  it('renders correctly with ariaValueText props set to String', () => {
    const tree = renderer.create(
      <Preloader kind='linear' ariaValueText='Some aria value' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with dynamicColor props set to true', () => {
    const tree = renderer.create(
      <Preloader kind='linear' dynamicColor />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with exactProgressLabel props set to true', () => {
    const tree = renderer.create(
      <Preloader kind='linear' exactProgressLabel />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with indeterminate props set to true', () => {
    const tree = renderer.create(
      <Preloader kind='linear' indeterminate />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with labelPositionX props set to one of enum value', () => {
    const tree = renderer.create(
      <Preloader kind='linear' labelPositionX='center' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with labelPositionY props set to one of enum value', () => {
    const tree = renderer.create(
      <Preloader kind='linear' labelPositionY='center' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with labelPostfix props set to String', () => {
    const tree = renderer.create(
      <Preloader kind='linear' labelPostfix='some string' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with max props set to Number', () => {
    const tree = renderer.create(
      <Preloader kind='linear' max={50} />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with min props set to Number', () => {
    const tree = renderer.create(
      <Preloader kind='linear' min={20} />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with progressLabel props set to true', () => {
    const tree = renderer.create(
      <Preloader kind='linear' progressLabel />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with secondaryProgressValue props set to Number', () => {
    const tree = renderer.create(
      <Preloader kind='linear' secondaryProgressValue={2} />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with subtitle props set to String', () => {
    const tree = renderer.create(
      <Preloader kind='linear' subtitle='Some string' />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with value props set to Number', () => {
    const tree = renderer.create(
      <Preloader kind='linear' value={2} />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with withSecondaryProgress props set to true', () => {
    const tree = renderer.create(
      <Preloader kind='linear' withSecondaryProgress />,
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});