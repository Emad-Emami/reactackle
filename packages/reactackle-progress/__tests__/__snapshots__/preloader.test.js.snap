// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preloader /> renders correctly with alignX prop set to one of enum values 1`] = `
<div
  className="ProgressHolderStyled-yJPzw cUkVRi"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with alignY prop set to one of enum values 1`] = `
<div
  className="ProgressHolderStyled-yJPzw cDzipC"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with ariaValueText props set to String 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    aria-valuetext="Some aria value"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with dynamicColor props set to true 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(255, 195, 15)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with exactProgressLabel props set to true 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="LabelWrapperStyled-eKVrkZ cMhIgn"
      >
        <label
          className="ValueLabelStyled-Lgqql NdteF"
        >
          0
        </label>
      </div>
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with height prop set to String 1`] = `
<div
  className="ProgressHolderStyled-yJPzw gWvmZs"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with indeterminate props set to true 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ fuWMLn"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
            }
          }
        >
          <div
            className="SplitterStyled-jQZnrY pPBcv"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with kind prop set to one of enum values 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with labelPositionX props set to one of enum value 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with labelPositionY props set to one of enum value 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with labelPostfix props set to String 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with max props set to Number 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="50"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with min props set to Number 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="20"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with progressLabel props set to true 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="LabelWrapperStyled-eKVrkZ cMhIgn"
      >
        <label
          className="ValueLabelStyled-Lgqql NdteF"
        >
          0%
        </label>
      </div>
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with secondaryProgressValue props set to Number 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with subtitle props set to String 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    aria-valuetext="Some string"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="LabelWrapperStyled-eKVrkZ cMhIgn"
      >
        <div
          className="SupplementTextStyled-gDaCZq NDsTy"
        >
          Some string
        </div>
      </div>
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with value props set to Number 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="2"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0.02)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with width prop set to Sring 1`] = `
<div
  className="ProgressHolderStyled-yJPzw dZhryn"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<Preloader /> renders correctly with withSecondaryProgress props set to true 1`] = `
<div
  className="ProgressHolderStyled-yJPzw hPaiJb"
  kind="linear"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    className="ProgressBoxStyled-ciJUOb enQWBS"
    role="progressbar"
  >
    <div
      className="LinearProgressStyled-bYugBt eXJaSf"
    >
      <div
        className="ContentWrapperStyled-dHfVBA llocGx"
      >
        <div
          className="SecondaryLineStyled-kfyfry eOWLqx"
          style={
            Object {
              "backgroundColor": "rgba(33, 150, 243, 0.15000000000000002)",
              "transform": "scaleX(0)",
            }
          }
        />
        <div
          className="PrimaryLineStyled-brPmnZ ehYOcW"
          style={
            Object {
              "backgroundColor": "rgb(33, 150, 243)",
              "transform": "scaleX(0)",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
